src/utils/registerServiceWorker.ts

```diff
@@ -1,4 +1,12 @@
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import connectConfig from 'connect-config';
+import path from 'path';
```

Comment:
Would prefer to avoid the path dependency here since it will run in the browser.

Maybe we can use `new URL(serviceWorkerScriptPath, connectConfig.routerBasename)`?